hikari/guilds.py

```diff
@@ -439,6 +440,51 @@ async def fetch_self(self) -> Member:
         """
         return await self.app.rest.fetch_member(self.guild_id, self.id)
 
+    async def ban(
+        self,
+        *,
+        delete_message_days: undefined.UndefinedOr[builtins.int] = undefined.UNDEFINED,
```

Comment:
don't need to import builtins here, just use `int` in the type hint (docstring type still has to be builtins.int though)

Reply:
@FasterSpeeding  i don't get it. Davfsa told me to put builtins.int there. Then if I put int in the type but builtins.int in docstring, that doesn't make any sense